eppo_core/src/ufc/models.rs

```diff
@@ -18,7 +18,8 @@ pub type Timestamp = chrono::DateTime<chrono::Utc>;
 pub(crate) struct UniversalFlagConfigWire {
     /// When configuration was last updated.
     pub created_at: Timestamp,
-    pub format: AssignmentFormat,
+    #[serde(default)]
```

Comment:
so we don't serialize `null`s:
```suggestion
    #[serde(default, skip_serializing_if = "Option::is_none")]
```